django-stubs/contrib/postgres/fields/array.pyi

```diff
@@ -27,7 +27,7 @@ class ArrayField(CheckFieldDefaultMixin, Field[_ST, _GT]):
         base_field: Field,
         size: Optional[int] = ...,
         *,
-        verbose_name: Optional[Union[str, bytes]] = ...,
+        verbose_name: Optional[Union[_StrOrPromise, bytes]] = ...,
```

Comment:
Not really in the scope of this PR, but I can't help to ask, why is `bytes` even allowed here? Makes no sense IMO. Is this for Python 2 compatibility?

The commit message adding this simply states "fixes" d43cb1fcd7961ca620aaf1d67949baf2e3c040f3 (?!)

Reply:
I think it might be some legacy thing, I am totally fine with removing it.

Reply:
OK I can open a PR for that after this is merged, if ljodal doesn't get to it first.

Reply:
I removed the bytes while fixing `name` :)